src/mainviews/job-detail.tsx

```diff
@@ -1,13 +1,398 @@
-import React from 'react';
-
-import { IJobDetailModel, JobsView } from '../model';
+import React, { useEffect, useState } from 'react';
+import { ICreateJobModel, IJobDetailModel, JobsView } from '../model';
+import Box from '@mui/material/Box';
+import Breadcrumbs from '@mui/material/Breadcrumbs';
+import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
+import Button from '@mui/material/Button';
+import Accordion from '@mui/material/Accordion';
+import {
+  AccordionDetails,
+  AccordionSummary,
+  Checkbox,
+  CircularProgress,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel,
+  OutlinedInputProps,
+  TextField,
+  Typography
+} from '@mui/material';
+import { caretDownIcon } from '@jupyterlab/ui-components';
+import { useTranslator } from '../hooks';
+import { Heading } from '../components/heading';
+import { Scheduler, SchedulerService } from '../handler';
+import { JupyterFrontEnd } from '@jupyterlab/application';
 
 export interface IJobDetailProps {
+  app: JupyterFrontEnd;
   model: IJobDetailModel;
   modelChanged: (model: IJobDetailModel) => void;
+  setCreateJobModel: (createModel: ICreateJobModel) => void;
   setView: (view: JobsView) => void;
 }
 
+interface ITextFieldStyledProps {
+  label: string;
+  defaultValue?: string;
+  InputProps?: Partial<OutlinedInputProps> | undefined;
+}
+
 export function JobDetail(props: IJobDetailProps): JSX.Element {
-  return <h1>Hi there!</h1>;
+  const [loading, setLoading] = useState(true);
+  const [job, setJob] = useState<Scheduler.IDescribeJob | undefined>(undefined);
+
+  //TO DELETE
+  const prop = { model: { jobId: '3f062962-1e3e-442b-8454-e76af250da39' } };
```

Comment:
Can you pull this from the model in the props?

Reply:
Removed, was using prop as a temporary clutch for testing